### ISSUE TITLE
Readme talks about using a release, but there are no releases in the page

### ISSUE THREAD

**Issue by qwertyuu** [MacPass/MacPassCSV#5]

**Comment by cyndi5** [comment]
@qwertyuu I was wondering the same thing. Where are the releases?

**Comment by mstarke** [comment]
There currently is no real release as the code is in a stale state until I get the time to finish it up. Export with a fixed data structure is already working but I wanted to add UI to change how and what to export in the CSV file.

**Comment by GittyGharial** [comment]
If it's stale will building from source be worthwhile?

**Comment by mstarke** [comment]
Nope. I was in a flow while implementing the plugin but my spare time got yanked so this is imcomplete. The code for export is more or less working. But I wanted to add a UI to choose the layout and enable/disable some fields and allow for some general options on the export format.

Import is not working at all currently.

**Comment by aktasberkin** [comment]
Hi, do you plan on finishing this project any time soon?
Appreciate the good work!
